gtsfm/densify/mvs_utils.py

```diff
@@ -44,6 +45,49 @@ def calculate_triangulation_angle_in_degrees(
     return geometry_utils.compute_relative_unit_translation_angle(Unit3(ray_1), Unit3(ray_2))
 
 
+def calculate_triangulation_angles_in_degrees(
```

Comment:
Do we need the non-vectorized function anywhere?

Reply:
Thanks for taking a look @ayushbaid. I'm using it in another branch, and wanted to keep the other branch lean. COLMAP relies pretty heavily on minimum triangulation angles -- we can discuss more tomorrow.